apps/ws/src/Game.ts

```diff
@@ -176,20 +193,20 @@ export class Game {
     this.gameId = game.id;
   }
 
-  async addMoveToDb(move: Move, moveTimestamp: Date) {
-    
+  async addMoveToDb(move: TimingMove) {
+    console.log(move);
     await db.$transaction([
       db.move.create({
         data: {
           gameId: this.gameId,
           moveNumber: this.moveCount + 1,
           from: move.from,
           to: move.to,
-          before: move.before,
-          after: move.after,
-          createdAt: moveTimestamp,
-          timeTaken: moveTimestamp.getTime() - this.lastMoveTime.getTime(),
-          san: move.san
+          before: move.before ?? '',
+          after: move.after ?? '',
+          createdAt: move.createdAt,
```

Comment:
shouldnt be client side
User will send a very old time to decrease their time and cheat
Needs to be set on the server

Reply:
Done